### PR TITLE
Facilitate local development in a Docker container

### PR DESCRIPTION
**Pull request by yankcrime**
**What this PR does / why we need it**:
This commit adds a few files that allow for local development using a Docker container instead.

### CODE REVIEW

**Review by meichstedt — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -15,6 +15,22 @@ yarn docs:dev
 
 The output will inform you about the address the site is served (usually http://localhost:8080).
 
+#### Via Docker
+You can choose to install and run the above dependencies in a Docker container instead, if you do not wish to pollute your local environment.
+
+First, build the development image with:
+
+``` bash
```
Comment (meichstedt) [inline README.md]:
```suggestion
```bash
```

File: README.md [hunk 2]
```diff
@@ -15,6 +15,22 @@ yarn docs:dev
...
+make dev-image
+```
+
+And then run VuePress in a container from this image by doing:
+
+``` bash
```
Comment (meichstedt) [inline README.md]:
```suggestion
```bash
```

**Review by meichstedt — COMMENTED** [review]
nit: it's unusual to put a space after a code block triple-backtick-sequence and before the language id. IDEA doesn't correctly syntax highlight the block content in this case.

File: README.md (hunk 1 above)
Comment (yankcrime) [reply README.md]: Good spot, thanks @meichstedt - blame my editor for doing that.

**Pull request merged**